src/main/java/com/laytonsmith/PureUtilities/Point3D.java

```diff
@@ -9,7 +9,7 @@
 	/**
 	 * A Point3D located at [0, 0, 0] (zero)
 	 */
-	public static final Point3D ZERO = new Point3D();
+	public static final Point3D ZERO = new Point3D(0, 0, 0);
```

Comment:
well, the reason I wanted to privatize Point3D() was so the only place it would be run was from this line. 
if you end up using Point3D(0, 0, 0), what's the point? nothing else in this class uses that constructor.